sqlserver/2017/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM mcr.microsoft.com/mssql/server:2017-CU22-ubuntu-16.04
+
+LABEL maintainer="Bonita R&D Runtime team <rd.runtime@bonitasoft.com>"
```

Comment:
Should that be public?

Reply:
Maybe put the list of dev in the runtime team?

Reply:
why, this is more generic like this if someone leaves or comes into the team

Reply:
After checking, we cannot access this address from the outside world, so I'll do as you suggest.

Reply:
fixed